docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -90,3 +90,14 @@ Your Airflow instance likely had specific IAM or Kubernetes permissions that all
 - **Using [Airflow connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) or [environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets)** are the easiest methods for defining permissions.
 
 - **If your Airflow DAGs used [`KubernetesPodOperators`](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html)**, it's possible that you used the `in_cluster` config or loading a `kube_config` file. When migrating, we recommend switching to [using connections with a `kube_config` JSON blob](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/connections/kubernetes.html) to make things easier.
+
+## Limitations
+
+Currently there are a few limitions of converting airflow DAGs into Dagster Jobs/Assets/Schedules:
```

Comment:
```suggestion
There are a few limitations to converting Airflow DAGs into Dagster Jobs, Assets, and Schedules. The following features are currently unsupported:
```

Some small grammar tweaks. Also, setting this up to cut down on some wordiness in the individual list items.